kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/server/Server.kt

```diff
@@ -37,12 +39,23 @@ class Server(
         request.body.utf8()
       )
       response.onNext(CallResponse(body = out.encodeUtf8()))
-    } catch (e: Throwable) {
+    } catch (t: Throwable) {
       response.onNext(CallResponse(error = CallResponse.Error(
-        message = (e.message ?: "internal error"),
-        stack = e.stackTraceToString())),
+        message = (t.message ?: t.stackTraceFirstLine()),
+        stack = t.stackTraceToString())),
```

Comment:
Probably better to only stringify the stack trace once and reuse it? I'm not sure how efficient it is to gather a stack but it seems like it could be quite slow.

Reply:
oof yes